bleak/__init__.py

```diff
@@ -8,13 +8,39 @@
 import os
 import sys
 import logging
-import platform
 import asyncio
 
 from bleak.__version__ import __version__  # noqa: F401
 from bleak.exc import BleakError
-
-_on_rtd = os.environ.get("READTHEDOCS") == "True"
+from bleak._api import (
+    BleakScanner,
+    _BleakScannerImplementation,
+    BleakClient,
+    _BleakClientImplementation,
+    BLEDevice,
+    _BLEDeviceImplementation,
+    BleakGATTServiceCollection,
+    BleakGATTService,
+    _BleakGATTServiceImplementation,
+    BleakGATTCharacteristic,
+    _BleakGATTCharacteristicImplementation,
+    BleakGATTDescriptor,
+    _BleakGATTDescriptorImplementation,
+    discover,
+)
+
+__all__ = [
+    "BleakError",
+    "BleakScanner",
+    "BleakClient",
+    "BLEDevice",
+    "BleakGATTServiceCollection",
+    "BleakGATTService",
+    "BleakGATTCharacteristic",
+    "BleakGATTDescriptor",
+    "discover",
+    "cli",
```

Comment:
`cli` has never really been developed or used AFAIK, so could probably be dropped.

Reply:
ok.